Assets/Mirror/Examples/AdditiveScenes/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using UnityEngine;
+
+namespace Mirror.Examples.Additive
+{
+    [RequireComponent(typeof(CharacterController))]
+    public class PlayerController : NetworkBehaviour
+    {
+        CharacterController characterController;
+
+        public float moveSpeed = 300f;
+        public float maxTurnSpeed = 90f;
+        public float turnSpeedAccel = 30f;
+        public float turnSpeedDecel = 30f;
+
+        public override void OnStartServer()
+        {
+            base.OnStartServer();
+            playerColor = Random.ColorHSV(0f, 1f, 1f, 1f, 0.5f, 1f);
+        }
+
+        [SyncVar(hook = nameof(SetColor))]
+        public Color playerColor = Color.black;
+
+        void SetColor(Color color)
+        {
+            GetComponent<Renderer>().material.color = color;
+        }
+
+        Camera mainCam;
+
+        public override void OnStartLocalPlayer()
+        {
+            base.OnStartLocalPlayer();
+
+            characterController = GetComponent<CharacterController>();
+
+            // Grab a refernce to the main camera so we can enable it again in OnDisable
+            mainCam = Camera.main;
+
+            // Turn off the main camera because the Player prefab has its own camera
+            mainCam.enabled = false;
+
+            // Enable the local player's camera
+            GetComponentInChildren<Camera>().enabled = true;
+        }
+
+        private void OnDisable()
+        {
+            if (isLocalPlayer)
+            {
+                // Disable the local player's camera
+                GetComponentInChildren<Camera>().enabled = false;
+
+                // Re-enable the main camera when Stop is pressed in the HUD
+                if (mainCam != null) mainCam.enabled = true;
+            }
+        }
+
+        float horiz = 0f;
+        float vert = 0f;
+        float turn = 0f;
+
+        void Update()
+        {
+            if (!isLocalPlayer) return;
+
+            horiz = Input.GetAxis("Horizontal");
```

Comment:
please use proper names like 'horizontal'. less magic :)

Reply:
Then I'll have to fix the Lobby controller too.

Reply:
full word names done as requested